package.json

```diff
@@ -1,16 +1,6 @@
 {
```

Comment:
Please run `npx sort-package-json` to sort this package.json. Stuff seems to have moved around

Reply:
The package.json has been cleaned up. 
Sonar-scanner will only run and update the board if a commit is pushed or a PR is created, so yes, you just need to consult the SonarCloud dashboard.